example-projects/post/sketches/space/index.js

```diff
@@ -0,0 +1,94 @@
+const { THREE } = window.HEDRON.dependencies
+const cubeTextureLoader = new THREE.CubeTextureLoader()
+
+class Space {
+  constructor (world) {
+    this.root = new THREE.Group()
+
+    world.camera.position.z = 40
+
+    world.scene.background = cubeTextureLoader
+      .setPath(`${__dirname}/assets/`)
+      .load([
+        'px.jpg', 'nx.jpg',
+        'py.jpg', 'ny.jpg',
+        'pz.jpg', 'nz.jpg',
+      ])
+
+    // Hacky way to get orbit controls working with the viewer
+    const viewerEl = document.getElementById('viewer')
+    this.controls = new THREE.OrbitControls(world.camera, viewerEl)
```

Comment:
Heh, I've been doing this for a while now! Ultimately this isn't ideal because eventually all of the rendering may get moved to a different thread where there is no access to the DOM and this tricky will no longer work. Not anytime soon though. I won't ever be changing the ID of that element.